src/VisualStudio/Core/Def/Implementation/VisualStudioRenameTrackingDismissService.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation
+{
+    /// <summary>
+    /// Handles dismissing the rename tracker when a symbol has changed
```

Comment:
... i feel like this needs a ton more information.  why woudl we need this *at all*?